src/Adapters/DatabasePendingApprovalRetriever.php

```diff
@@ -58,7 +58,7 @@ private function queryPendingApprovals( array $categories ): IResultWrapper {
 				'latest_approval.al_timestamp'
 			],
 			[
-				'cl_to' => $categories,
+				'cl_to' => $this->normalizeCategoryTitles( $categories ),
```

Comment:
The Retriever gets the categories from the ApproverRepository, which now returns the human-readable names. It has to be normalized to DBKey form here since it is querying the database.